src/main/kotlin/build/buf/intellij/resolve/BufRootsProvider.kt

```diff
@@ -62,6 +72,10 @@ class BufRootsProvider : AdditionalLibraryRootsProvider() {
                 ?.findFileByRelativePath("v2/module")
         }
 
+        fun getIDEAModCacheV2(): Path {
+            return Paths.get(PathManager.getSystemPath(), "buf", "modcache")
```

Comment:
Now storing in `~/Library/Caches/<product>/buf/modcache` so if we ever want to put other stuff in the `buf` toplevel folder we can.